pkg/controller/dr/drcontroller.go

```diff
@@ -208,9 +215,11 @@ func (d *DrController) DisableReplication(ctx *c.Context, replica *ReplicationSp
 	if err != nil {
 		return err
 	}
-	err = d.secondaryOp.Disable(ctx, replica, secondaryVol)
-	if err != nil {
-		return err
+	if primaryVol.Id != secondaryVol.Id {
```

Comment:
why  ? for all cases where the check is added for primaryVol.Id != secondaryVol.Id